Runtime/McSubscriber.cs

```diff
@@ -508,6 +553,35 @@ public void RemoveRenderAudioSource(AudioSource source)
         {
             _renderer.RemoveAudioTarget(source);
         }
+        private Layer GetSimulcastlayer(string rid)
+        {
+            foreach (var item in simulCastInfo.Layers)
+            {
+                if(rid.ToLower().Equals(item.EncodingId.ToLower()))
+                    return item;
+            }
+            return null;
+        }
+        /// <summary>
+        /// Set a simulcast layer
+        /// </summary>
+        /// <param name="layer">
+        public void SetSimulcastLayer(string simulcastId)
+        {
+            Layer layer = GetSimulcastlayer(simulcastId);
+            if(layer != null)
+            {
+                var layerpayload = new Dictionary<string, dynamic>();
```

Comment:
can we get rid of the use of dynamic here for backwards compatibility? This will force clients to use a new version of C#.